docs/DeveloperGuide.adoc

```diff
@@ -782,6 +782,7 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
+* NUS Students living in RC
```

Comment:
Is it better to write as `Residential College (RC)` since other developers might not know what `RC` stands for?